api/v1beta1/zz_generated.openapi.go

```diff
@@ -673,15 +673,13 @@ func schema_sigsk8sio_cluster_api_api_v1beta1_ClusterStatus(ref common.Reference
 					"infrastructureReady": {
 						SchemaProps: spec.SchemaProps{
 							Description: "InfrastructureReady is the state of the infrastructure provider.",
```

Comment:
Well that's fun:

> Default values can be specified on a field using the +default= tag. Primitives will have their values directly assigned while structs will go through the JSON unmarshalling process. Fields that do not have an omitempty json tag will default to the zero value of their corresponding type if no default is assigned.
> https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

I assume that is something that is only relevant for core types. Our generated OpenAPI types are **not** used by Kubernetes and the apiserver doesn't know anything about our Go types.

So the change in this generated file should have zero impact.